homeassistant/components/suez_water/strings.json

```
@@ -6,7 +6,13 @@
           "username": "[%key:common::config_flow::data::username%]",
           "password": "[%key:common::config_flow::data::password%]",
           "counter_id": "Counter id"
-        }
+        },
+        "data_description": {
+          "username": "Enter your login associated with your Tout sur mon eau account",
```

Is it possible to use a placeholder for "Tout sur mon eau"?
(I'm not sure if it's possible for data description, but if it does that would be better)

---

It is something that should not be translated, so I'm thinking about something like this:
```suggestion
          "username": "Enter your login associated with your {tout_sur_mon_eau} account",
```

---

it's done and its working.
I used the placeholder in description and data_description